BangazonTerminalInterface/Controllers/PaymentController.cs

```diff
@@ -8,5 +9,31 @@ namespace BangazonTerminalInterface.Controllers
 {
     class PaymentController
     {
+        string paymentType; //To Capture the payment type
+        int paymentAccountNumber; //To capture the account number
+        string newPayment; //To build the new payment that will be added to the DB
+        
+        private void requestPaymentType() // Ask for payment Type
```

Comment:
yes, I heard you guys talking last night about a helper function, that makes a lot of sense ... will refactor!